acceptance-tests/src/test/resources/features/site/datasets.feature

```diff
@@ -53,3 +53,15 @@ Scenario: Display multiparagraph summary
         | Mauris ex est, dapibus in dictum ut, elementum sit amet odio. Proin ...       |
         | Cras fringilla odio sit amet tellus pellentesque posuere. Etiam semper ...    |
         | Curabitur quis eros nisi. Nulla leo est, elementum non gravida suscipit ...   |
+
+Scenario: Show new v3 files with text along with old v2 files
```

Comment:
Reference to this scenario needs to be added to the JIRA ticket referencing all bits of code that need removing once the data upgrade scripts are run (see comment in the review of `Publication.java#getAttachmentsOld`).

Reply:
Created RPS-238